### DISCUSSION TITLE
What does the feedback function do?

### DISCUSSION THREAD

**Discussion by Tne-5** [open-webui/open-webui#2810] [General]
![image](https://github.com/open-webui/open-webui/assets/62752844/9aa3e3fd-2283-458d-aaaf-a70faf5109cb)

What I'm hoping is that the feedback is somehow used to fine-tune whichever language model you are using, is this the case? Thanks.

**Comment by justinh-rahb** [comment]
Not automatically, no. All these feedback and annotations are stored in the chats database and could in theory be exported and processed into RLHF training data by administrators, but at the moment that's left entirely as an excercise for you as the end-user. We just provide the means to add metadata, we're not actually utilizing it.

**Reply by seanpham99** [reply]
Does it means you guys do not have the function to save the feedback into the db right?
